lib/ExtendTruffleContract.ts

```diff
@@ -106,97 +106,92 @@ export abstract class ExtendTruffleContract {
 
   public get address() { return this.contract.address; }
 
+  /**
+   * Return a function that creates an EventFetcher<TArgs>.
+   * For subclasses to use to create their event handlers.
+   * This is identical to what you get with Truffle, except that
+   * the result param of the callback is always guaranteed to be an array.
```

Comment:
@tibetsprague  Do you refer to this:

> the result param of the callback is always guaranteed to be an array ?

It was because without that then one always has to write a stupid bit of code in the event handler
to check whether or not the value is an array.  Since that code always has to handle the case of being an array, it shouldn't be a problem for anyone.  If in the much more rare case that they want to know if there is only one item, they can check the length of the array.  But I don't think many people will be concerned about this, more likely they will be happy that their code can be  cleaner and simpler.